main.cc

```diff
@@ -420,12 +420,33 @@ int main(int ac, char** av) {
     // If --version is requested, print it out and exit immediately to avoid
     // Seastar-specific warnings that may occur when running the app
     bpo::variables_map vm;
-    bpo::store(bpo::command_line_parser(ac, av).options(app.get_options_description()).allow_unregistered().run(), vm);
+    auto parsed_opts = bpo::command_line_parser(ac, av).options(app.get_options_description()).allow_unregistered().run();
+    bpo::store(parsed_opts, vm);
     if (vm["version"].as<bool>()) {
         fmt::print("{}\n", scylla_version());
         return 0;
     }
 
+    fmt::print("Scylla version {} starting ...\n", scylla_version());
+    if (ac) {
+        fmt::print("command used: \"{}", av[0]);
+        for (int i = 1; i < ac; ++i) {
+            fmt::print(" {}", av[i]);
+        }
+        fmt::print("\"\n");
+    }
+
+    fmt::print("Parsed command line options: [{}]\n",
+        boost::algorithm::join(parsed_opts.options | boost::adaptors::transformed([] (const auto& opt) {
+            if (opt.value.empty()) {
```

Comment:
Please avoid using generic lambdas when not needed, generic lambdas makes the types not accessible to IDEs and make error messages worse.

Reply:
Done.